Copy to SD Card root directory to update/Language Packs/language_hu.ini

```diff
@@ -1,312 +1,312 @@
+label_recover_feedrate:Előtolás javítás
+label_swap_recover_feedrate:Csere előtolás jav.
+label_mbl_offset:MBL Eltolás
+label_save:Mentés
+label_restore:Visszaállítás
+label_reset:Alapállítás
+label_eeprom_save_info:Mented a megváltozott\nbeállításokat az EEPROM-ba?
+label_eeprom_restore_info:Betöltöd a beállításokat az\nEEPROM-ból?
+label_eeprom_reset_info:Visszaállítod az EEPROM-ot a\ngyári beállításokra?
+label_settings_save:Beállítások mentése
+label_settings_restore:Beállítások újratöltése
+label_settings_reset:Beállítások visszaáll.
+label_settings_reset_info:Minden beállítás vissza\nfog állni alapértelmezett\nértékre. Folytatod?
+label_settings_reset_done:A beállítások visszaállítása\nsikeresen megtörtént.\nAz aktiváláshoz indítsd\nújra az eszközt.
+label_page_up:Oldal fel
+label_page_down:Oldal le
+label_up:Fel
+label_down:Le
+label_next:Tovább
+label_back:Vissza
+label_inc:Növel
+label_dec:Csökkent
+label_load:Betöltés
+label_unload:Kiadás
+label_on:BE
+label_off:KI
+label_auto:AUTO
+label_smart:OKOS
+label_slow:Lassú
+label_normal:Normál
+label_fast:Gyors
+label_zero:Nulla
+label_half:Fél
+label_full:Teljes
+label_custom:Egyedi
+label_clear:Törlés
+label_default:Alapért.
+label_start:Kezdés
+label_stop:Állj
+label_pause:Szünet
+label_resume:Folytatás
+label_init:Kezdet
+label_disconnect:Leválaszt
+label_shut_down:Leállítás
+label_force_shut_down:Kényszerít
+label_emergencystop:Vész-STOP
+label_preheat:Előfűtés
+label_preheat_both:Mindkettő
+label_cooldown:Hűtés
+label_confirm:Oké
+label_cancel:Mégse
+label_warning:Figyelem!
+label_continue:Folytatás
+label_confirmation:Biztos vagy benne?
+label_status:Státusz
+label_ready:Kész
+label_busy:Feldolgozás alatt, kérlek várj...
+label_loading:Betöltés...
+label_unconnected:Nincs nyomtató csatlakoztatva!
+label_info:Infó
+label_invalid_value:Nincs érvényes érték megadva
+label_timeout_reached:Időkorlát elérve!
+label_disconnect_info:Most a számítógépéről vezérelheted a nyomtatót!
+label_shutting_down:Leállítás...
+label_wait_temp_shut_down:Várd meg a fej\nvisszahűlését.%d℃
+label_power_failed:Folytatod a nyomtatást?
+label_process_running:Folyamat fut!
+label_process_completed:Folyamat befejezve!
+label_process_aborted:Folyamat megszakítva!
+label_tftsd:TFT-SD
+label_read_tftsd_error:TFT-SD olvasási hiba!
+label_tftsd_inserted:SD-Kártya behelyezve!
+label_tftsd_removed:SD-Kártya eltávolítva!
+label_u_disk:USB Tár
+label_read_u_disk_error:USB olvasási hiba!
+label_u_disk_inserted:USB behelyezve!
+label_u_disk_removed:USB eltávolítva!
+label_onboardsd:Lap-SD
+label_read_onboardsd_error:Alaplap-SD olvasási hiba!
+label_filament_runout:Szál kifutás!
+label_disable_steppers:Mot.tilt
+label_xy_unlock:X-Y Felold
+label_start_print:%s\nfájl kiválasztva.\nElindítod a nyomtatását?
+label_stop_print:Megállítod a nyomtatást?
+label_is_pause:A művelet tiltva van\na nyomtatás alatt.\nSzünetelteted a nyomtatást?
+label_m0_pause:Szünetelés M0 parancsal
+label_test:Teszt
+label_deploy:Telepítés

```

Comment:
With all due respect I have to make a comment because the translation is wrong.
Deploy means to move to a point and be ready for action. The word you used in Hungarian means "to install". The label you translated doesn't refer to installing the BLTouch but to make it ready for measurements.
If you do not like the way I made corrections, no problem, but please use synonyms, not words with a different meaning than they are used in the TFT's FW.